lib/msal-common/src/response/AuthenticationResult.ts

```diff
@@ -44,4 +44,6 @@ export type AuthenticationResult = {
     msGraphHost?: string;
     code?: string;
     fromNativeBroker?: boolean;
+    managedIdentityClientId?: string;
+    managedIdentityResource?: string;
```

Comment:
Do we need to return these? The user already knows what they are since they pass them into a request.

Reply:
We do not need the managedIdentityClientId but pass the managedIdentityResource as scopes on line 29.

Reply:
Also this is a public type, any changes to public API should be highlighted and reviewed. 

In this case, the change is not needed. AuthResult does not need to change for MSI.